corehq/apps/export/forms.py

```diff
@@ -102,8 +102,13 @@ class DateSpanField(forms.CharField):
     def clean(self, value):
         date_range = super(DateSpanField, self).clean(value)
         dates = date_range.split(DateRangePickerWidget.separator)
-        startdate = dateutil.parser.parse(dates[0])
-        enddate = dateutil.parser.parse(dates[1])
+        if len(dates) != 2:
+            return
+        try:
+            startdate = dateutil.parser.parse(dates[0])
+            enddate = dateutil.parser.parse(dates[1])
+        except ValueError:
+            raise ValueError(_("Date range invalid. Pleaes select a valid date range."))
```

Comment:
I don't know. All I wanted to do here was change the message which get's displayed to the user.